Sources/DeepLearning/Random.swift

```diff
@@ -125,6 +125,10 @@ private typealias UInt32x4 = (UInt32, UInt32, UInt32, UInt32)
 /// share state. The random data generated is of high-quality, but is not
 /// suitable for cryptographic applications.
 public struct ThreefryRandomNumberGenerator: SeedableRandomNumberGenerator {
+    public static var global = ThreefryRandomNumberGenerator(
+        uint64Seed: ARC4RandomNumberGenerator.global.next()
```

Comment:
Sorry for speculation since I'm not familiar with the code.
Is depending on `ARC4RandomNumberGenerator.global.next()` as the seed intentional here?

Reply:
Yeah. Now that I think about it, though, I don't know if it's a good idea. The original intent was to seed the PRNG with a 64-bit value (as opposed to the 32-bit value returned by `time(nil)`). However, doing this doesn't actually buy us anything, since arc4random is also deterministic. I'll change it.